tsconfig.base.all.json

```diff
@@ -252,7 +258,9 @@
       "@fluentui/recipes": ["packages/react-components/recipes/src/index.ts"],
       "@fluentui/theme-designer": ["packages/react-components/theme-designer/src/index.ts"],
       "@fluentui/tokens": ["packages/tokens/src/index.ts"],
-      "@fluentui/workspace-plugin": ["tools/workspace-plugin/src/index.ts"]
+      "@fluentui/workspace-plugin": ["tools/workspace-plugin/src/index.ts"],
+      "react-color-picker-1-preview": ["packages/react-components/react-color-picker-1-preview/src/index.ts"],
```

Comment:
this seems wrong, was this added by the generator ?

Reply:
ran `yarn nx g @fluentui/workspace-plugin:tsconfig-base-all`, problem is fixed :)